docs/configuration/backends/kubernetes.md

```diff
@@ -156,7 +156,8 @@ The following general annotations are applicable on the Ingress object:
 | `traefik.ingress.kubernetes.io/redirect-regex: ^http://localhost/(.*)`          | Redirect to another URL for that frontend. Must be set with `traefik.ingress.kubernetes.io/redirect-replacement`.                               |
 | `traefik.ingress.kubernetes.io/redirect-replacement: http://mydomain/$1`        | Redirect to another URL for that frontend. Must be set with `traefik.ingress.kubernetes.io/redirect-regex`.                                     |
 | `traefik.ingress.kubernetes.io/rewrite-target: /users`                          | Replaces each matched Ingress path with the specified one, and adds the old path to the `X-Replaced-Path` header.                               |
-| `traefik.ingress.kubernetes.io/rule-type: PathPrefixStrip`                      | Override the default frontend rule type. Default: `PathPrefix`.                                                                                 |
```

Comment:
Should we leave a note that `ReplacePath` is deprecated?

Reply:
Not really. If it is "deprecated" and we support it, whats the point of saying its deprecated...

Reply:
You could argue that we should phrase the log message even stronger: that the old way _will_ not be deprecated but _is_ from now on effectively. This means that we will still support it but plan to remove support in a future release. At least, that's what I remember we (in particular @ldez) did in the past to migrate options.

Reply:
I have completely remove annotations with the previous "label" style from the documentation.
I think can do the same here.